var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -234,3 +234,31 @@
     line: '{{ item.key }}="{{ item.value }}"'
   with_items:
     - { key: 'JAVA_HOME', value: '/usr/lib/jvm/oracle-java8-jdk-amd64' }
+
+#
+# Configure the Azure agent. Only run this on Azure, since that is the
+# only platform that has the Azure agent installed.
+#
+- lineinfile:
+    dest: /etc/waagent.conf
```

Comment:
This is more a question on our ansible style -- should we be using `path` or `dest` here?

Reply:
Looking [here](https://docs.ansible.com/ansible/latest/modules/lineinfile_module.html), it does seem like `dest` has been replaced with `path`, but `dest`, `destfile`, and `name` all still work. I'd opt for moving to `path` since that seems like the approach the docs imply is best, and it might save us some work later if/when the aliases are removed.